web/src/components/wizard/config/ConfigurationStep.tsx

```diff
@@ -141,12 +142,22 @@ const ConfigurationStep: React.FC<ConfigurationStepProps> = ({ onNext }) => {
             id={item.name}
             label={item.title}
             value={item.value || ''}
-            placeholder={item.default}
```

Comment:
Why remove this?

Reply:
see discussion above with Alex

Reply:
https://github.com/replicatedhq/embedded-cluster/pull/2447#discussion_r2196280441

Reply:
i actually just tested with KOTS, and it does use the default as the placeholder. but like i said above, i do see online that that's an anti-pattern because it's not what placeholders are intended for, so i still support showing the default value beneath the text area instead.